### DISCUSSION TITLE
cogVLM运行demo会报错

### DISCUSSION THREAD

**Discussion by wanghq09** [THUDM/CogVLM#340] [Q&A] [answered]
cogVLM仓库中只有zip文件，解压后是megetron的mp_rank.pt，尝试用代码demo直接按目录加载模型会报错，找不到safetensor，求助，这个问题是如何解决的?另外，模型路径上的readme是cli_demo.py，github的readme是cli_demo_hf.py

**Comment by wanghq09** [comment]
hf格式的模型有单独的仓库，在hugging-face上，地址为https://huggingface.co/THUDM/cogvlm-chat-hf，国内不方便下载，可以同步到其他两个平台上么

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
魔搭有
